spec/features/review_card_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe "the review card process" do
+  let!(:card) { create(:card) }
+
+  it "opens home page" do
+    visit root_path
+    expect(page).to have_content "Карточка"
+  end
+
+  it "true review card" do
+    visit root_path
+    fill_in :user_input, with: "Home"
+    click_button 'Проверить'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.